mips/ZeldaTables.py

```diff
@@ -34,8 +34,10 @@ def __init__(self, filename, vromStart, vromEnd, romStart, romEnd, sizeVrom, com
         self.number = number
 
 
-def getFileAddresses(filePath: Path | None) -> dict[str, FileAddressesEntry]:
-    table: dict[str, FileAddressesEntry] = dict()
+
+
+def getFileAddressList(filePath: Path | None) -> list[FileAddressesEntry]:
+    result: list[FileAddressesEntry] = []
```

Comment:
I'm curious on why you changed this instead of using `.values()` on the dict

Reply:
I needed to get the next entry for animal forest overlays, and while technically Python dictionaries are now always sorted by insertion order it still feels weird to depend on that, idk